.circleci/config.yml

```diff
@@ -10,6 +10,12 @@ jobs:
             bundler_version:
                 type: string
                 default: 2.2.29
+            postgres_version:
+                type: string
+                default: 9.6.2-alpine
```

Comment:
I'm not seeing where we're specifying Postgres 13.x anywhere else that would override this.